docs/output.md

```diff
@@ -168,7 +168,7 @@ newly recovered components are stored in a file (components_recovered.csv).
 - `pixelator`
 
   - `annotate`
-    - `<sample-id>.dataset.pxl`
+    - `<sample-id>.annotated.dataset.pxl`
     - `<sample-id>.meta.json`: Command invocation metadata.
     - `<sample-id>.rank_vs_size.png`
     - `<sample-id>.raw_components_metrics.csv`
```

Comment:
These are csv.gz instead

Reply:
Good catch! I will update all this things in an incoming PR.